install.sh

```diff
@@ -115,6 +125,10 @@ while [ -n "$CAP_ADD_PARAM" ] ; do
 done
 
 if [ -n "$IP_ADDRESS_PARAM" ] ; then
+	if [[ -n "$NET_PARAM" && "$NET_PARAM" != "host" ]]; then
+		echo "--ip $IP_ADDRESS_PARAM" >> "$HOST$DATADIR"/docker-run-opts
+		OPTS="$OPTS --ip $IP_ADDRESS_PARAM"
+	fi
```

Comment:
I'm not sure about this one and would prefer it was handled in different pull request. The goal of the `$IP_ADDRESS_PARAM`/`IPA_SERVER_IP` is to tell the software in the container what IP address it can assume to be available. Not set the IP address of the container.

Reply:
I've removed the part responsible for setting the container ip address. I'll open a separate pr as suggested.